### DISCUSSION TITLE
How to contribute, such as fixing bugs, etc.

### DISCUSSION THREAD

**Discussion by zghlx** [dbeaver/dbeaver#21689] [Q&A]
I've been able to compile the code correctly.
How to contribute, such as fixing bugs, etc. Can you provide a detailed guide?

**Comment by LonwoLonwo** [comment]
Hello @zghlx 

You can find some helpful info here: https://github.com/dbeaver/dbeaver/wiki/Contribute-your-code

**Comment by zghlx** [comment]
Is there any other IM group to communicate develop idea/problem/etc

**Reply by LonwoLonwo** [reply]
You can ask your questions on the GitHub. Our development team is in touch.
